### PR TITLE
 OZ-49: Refined setup instructions for Linux.

### PR DESCRIPTION
**Pull request by kdaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Ozone FOSS requires you to log into each component separately:
 ## 3. Analytics in Ozone FOSS
 There is a number of manual steps required to refresh analytics data in Ozone FOSS, please follow [this](readme/analytics.md) guide to know more.
 
-:bulb: **Did you know?** Ozone Pro embeds a service called *Ozone Analytics* that streams data from all its components automatically and in real-time data to any reporting and BI platform.
+:bulb: **Did you know?** Ozone Pro embeds a service called *Ozone Analytics* that streams data from all its components automatically and in real-time transfers data to any reporting and BI platform.
```
Comment (kdaud) [inline README.md:49]: `Self review:` I suppose the Ozone Analytics after streaming data from all it's component, it then **transfers** the data to any reporting and BI platform

File: README.md [hunk 2]
```diff
@@ -13,7 +13,10 @@ This Docker Compose project makes it easy to start and run an Ozone FOSS server
 Install Git, Maven, Docker and Docker compose.
 
 ### Run Ozone FOSS locally
-Create your working directory and as a root user:
+Install Git, Maven and Docker, open a terminal and type in the following:
```
Comment (kdaud) [inline README.md:16]: I think we can get rid of the above change since it's already covered under the section of `Pre-requisite `

**Comment by kdaud** [comment]
FWIW: The local installation goes on well with different versions of docker and  docker compose including the latest versions except docker version <20.10.13

**Comment by kdaud** [comment]
@Ruhanga basing on what have tested out, I suggest in our **Note** section we use `sudo -s` instead of `sudo -i`

File: readme/manual-setup.md [hunk 3]
```diff
@@ -5,18 +5,22 @@ Welcome to the Ozone FOSS manual setup guide. This guide details the setup of Oz
 ### Create your working directory
 
 Move to the location of your choice, e.g., your home folder:
+
+Note: On Linux OS you'll have to open the terminal as root/admin user by typing `sudo -i` and then entering correct user's password if prompted
```
Comment (mks-d) [inline readme/manual-setup.md:9]: @kdaud put this at the moment when it is really necessary. When exactly does the user start needing to be elevated with `sudo`?

Comment (kdaud) [reply readme/manual-setup.md:9]: The docker compose fails the run for normal users(Here are samples of [logs](https://talk.openmrs.org/t/ozone-installation/38770/4), so the user must be elevated with privileges.

Comment (kdaud) [reply readme/manual-setup.md:9]: From what have tested out, I would recommend we use `sudo -s` not `sudo -i`. I would need @Ruhanga's advice on this before I go a head to update the PR.

Comment (kdaud) [reply readme/manual-setup.md:9]: > When exactly does the user start needing to be elevated with sudo?

At the start of the process after creating the working directory

Comment (kdaud) [reply readme/manual-setup.md:9]: The ideal of elevating the user with `sudo` applies for both the Quick start guide and the manual setup guide.

Comment (Ruhanga) [reply readme/manual-setup.md:9]: @kdaud, `sudo -s` is okay. Could you go ahead to add that change...

File: readme/manual-setup.md [hunk 4]
```diff
@@ -67,7 +65,7 @@ $ export DISTRO_PATH=/your/path/to/ozone-distro/target/ozone-distro-1.0.0-SNAPSH
 
 ### Start Ozone
 ```bash
-$ docker compose -p $DISTRO_GROUP up
+$ sudo -E docker compose -p $DISTRO_GROUP up
 ```
 
-**Important:** This assumes that you run the `docker` command as the same user and in the same window in which you exported your variables. The variables will not be defined if Docker is run as `sudo`. Make sure to either export them as `root`, or run `docker` with `sudo -E` option to preserve the user environment. See also ['Post-installation steps for Linux'](https://docs.docker.com/engine/install/linux-postinstall/) for more details.
```
Comment (kdaud) [inline readme/manual-setup.md:69]: The docker compose runs successfully when the environment variables are exported as normal user so we remove the instruction of exporting them as a root. cc: @mks-d @Ruhanga

**Review by mks-d — COMMENTED** [review]
I pushed a bunch of changes on your fork and left a comment.

File: readme/manual-setup.md [hunk 5]
```diff
@@ -5,18 +5,22 @@ Welcome to the Ozone FOSS manual setup guide. This guide details the setup of Oz
...
 ```bash
 $ export OZONE_DIR=$PWD/ozone && \
 mkdir -p $OZONE_DIR && cd $OZONE_DIR
 ```
-### Clone the docker-compose project
+### Clone the ozone-docker project
 
 ```bash
-$ git clone https://github.com/ozone-his/ozone-docker
+$ git clone https://github.com/ozone-his/ozone-docker.git
```
[the remark this answers is not in the document]
Comment (kdaud) [reply readme/manual-setup.md]: Sure, [this doc](https://gitprotect.io/blog/how-to-clone-using-https-in-git/#:~:text=And%20that%E2%80%99s%20all,also%20considered%20valid.) points it out. Have fixed it in the subsequent commit.

File: readme/manual-setup.md [hunk 6]
```diff
@@ -64,8 +64,26 @@ $ export DISTRO_PATH=/your/path/to/ozone-distro/target/ozone-distro-1.0.0-SNAPSH
 ```
 
 ### Start Ozone
+<table>
+<tr>
+<td> macOS </td> <td> Linux </td>
+</tr>
+<tr>
+<td>
```
Comment (kdaud) [inline readme/manual-setup.md:72]: It's a great suggestion to add a table for Mac and Linux os, should we also think of the Window os?

**Review by jcarney2024 — APPROVED** [review]
This table provides much needed information for Linux users. Before this, just executing the startup file would throw errors, whereas running it with sudo -E successfully started up Ozone! Thank you!

**Review by Ruhanga — APPROVED** [review]
LGTM

**Pull request merged**